libpod/events/events.go

```diff
@@ -29,6 +49,96 @@ type Event struct {
 	Type Type
 }
 
+// EventerOptions ...
+type EventerOptions struct {
+	// EventerType describes whether to use journald or a file
+	EventerType string
+	// LogFilePath is the path to where the log file should reside if using
+	// the file logger
+	LogFilePath string
+}
+
+// Eventer is the interface for journald or file event logging
+type Eventer interface {
+	// Write an event to a backend
+	Write() error
+	// Read an event from the backend
+	Read(options ReadOptions) error
+}
+
+// ReadOptions describe the attributes needed to read event logs
+type ReadOptions struct {
+	// EventChannel is the comm path back to user
+	EventChannel chan *Event
+	// Filters are the results from the cli for --filter
+	Filters []string
```

Comment:
This should be functions - look at how we filter images and containers

Reply:
...Can we not do that because journald? Damn it...

Reply:
agreed to defer to do this right, i..e to a map